README.md

```diff
@@ -1,75 +1,84 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Doctors Appointment Front-end
```

Comment:
- [ ] Your frontend project is beautiful.  Let's show it off to your visitors with a screenshot.  That way, your users will know what your app looks like. 👍